examples/vision/md/supervised-contrastive-learning.md

```diff
@@ -8,8 +8,28 @@
 
 <img class="k-inline-icon" src="https://colab.research.google.com/img/colab_favicon.ico"/> [**View in Colab**](https://colab.research.google.com/github/keras-team/keras-io/blob/master/examples/vision/ipynb/supervised-contrastive-learning.ipynb)  <span class="k-dot">•</span><img class="k-inline-icon" src="https://github.com/favicon.ico"/> [**GitHub source**](https://github.com/keras-team/keras-io/blob/master/examples/vision/supervised-contrastive-learning.py)
 
+## Introduction
 
+[Supervised Contrastive Learning](https://arxiv.org/abs/2004.11362)
+(Prannay Khosla et al.) is a training methodology that outperforms
+plain crossentropy-supervised training on classification tasks.
```

Comment:
May I just suggest a few small things here:

- "Plain... training" may appear subjective, so I'd drop "plain" because it doesn't change the underlying meaning.
- "Cross-entropy"

```suggestion
supervising training on classification tasks with cross-entropy.
```

WDYT?

Reply:
LGTM